_posts/2019-07-18-edge-to-kubernetes.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+current: post
+cover: hacker-in-the-dark.jpg
+title: Connecting Edge IoT devices with Kubernetes
+date: 2019-07-18 08:00:00
+category: tutorials
+author: Viktor Petersson
+tags: [Open Source]
+class: post-template
+---
+
+# Bridging Edge IoT devices with Kubernetes
+
+**Required Skill Level**: Medium to Expert
+
+In my [last blog post]({{site.url}}/blog/tutorials/2019/07/15/mtls-with-nginx), we covered how to configure Nginx to use Mutual TLS (mTLS). This illustrated how mTLS can both simplify the back-end, while also make it more secure. In this article, we will take this one step further and integrate this with [Kubernetes](https://kubernetes.io/). If you haven't read the last blog post, that would be a good place to start.
+
+## Pre-requisites
+
+Before we dive into the actual configuration, we will assume that you have the following already configured:
+
+ * A Kubernetes cluster (d'oh!)
```

Comment:
I would maybe actually add in the Intro something about being familiar with Kubernetes to begin with because this means nothing to me

Reply:
Yeah good point. Will add.